### ISSUE TITLE
Opening tif returns Error code: 500 

### ISSUE THREAD

**Issue by helkejaa** [OCR4all/LAREX#216]
Opening a book which contains at least one tif file results in error code 500. I'm quite unsure whether this is a feature of the system or a problem in my own system. Opening png or jpg works fine. For comparison (although possibly not comparable feature), having an xml corresponding a picture file in alto format results in the book opening but with buffering sign on the relevant picture (that is, book opens, no 500).

Otherwise I love the system. Handling tifs should be relevant since the format is widely used in various scanning and photo equipment.

**Comment by maxnth** [comment]
>I'm quite unsure whether this is a feature of the system or a problem in my own system

It's indeed a problem with LAREX, I was able to reproduce it as well.
This is most likely happening because ImageIO for Java 8 doesn't natively support reading TIFF files. Starting with Java 9 TIFF support was added but as we're most likely going to migrate from Java 8 to Java 17 sometime in the future (so jumping to Java 9 isn't really a viable option for us) I'll look into adding TIFF support some other way (e.g. using JAI)

> having an xml corresponding a picture file in alto format results in the book opening but with buffering sign on the relevant picture

We currently only support PAGE XML so LAREX can't load any annotations from the ALTO file and this leads the infinite buffering sign. Adding some kind of warning in cases like this and the possibility to discard "invalid" XML files in the GUI would probably make sense, we'll look into it.
